src/briefcase/cmdline.py

```diff
@@ -36,19 +35,18 @@
 ]
 
 
-def parse_cmdline(args):
+def parse_cmdline(args, console: Console = Console()):
```

Comment:
The use of the default here means a Console() instance will always be created (as the default argument is evaluated when the function is defined). Passing in None, and constructing if the argument is None would be preferable.

Reply:
good point; defaulting to `None` now